info.yml

```diff
@@ -21,7 +21,7 @@ display_name: "Shotgun Utilities Framework"
 description: "A Collection of QT objects and utilities to simplify app development and Shotgun connectivity."
 
 # Required minimum versions for this item to run
-requires_shotgun_version: "v7.0.2"
+requires_shotgun_version:
```

Comment:
this is confusing. Are we limiting all of the framework to require server 7.0.2 and at the same time adding a graceful fallback in case the server isn't 7.0.2?